diff --git a/dist/runtime/database/server/utils/migrations/remote.js b/dist/runtime/database/server/utils/migrations/remote.js
index 0cec3738637f5cf78305ca2883507afbbd5a276b..7d9287793b629706856c4fdf9046a985584a2ad9 100644
--- a/dist/runtime/database/server/utils/migrations/remote.js
+++ b/dist/runtime/database/server/utils/migrations/remote.js
@@ -63,7 +63,8 @@ export async function applyRemoteDatabaseQueries(hub) {
   if (!queriesPaths.length) {
     return true;
   }
-  for (const queryPath of queriesPaths) {
+  
+  for await (const queryPath of queriesPaths) {
     let query = await queriesStorage.getItem(queryPath);
     if (!query) continue;
     if (query.replace(/\s$/, "").at(-1) !== ";") query += ";";
@@ -74,6 +75,6 @@ export async function applyRemoteDatabaseQueries(hub) {
       return false;
     }
     consola.success(`Database query \`${queryPath}\` applied`);
-    return true;
   }
+  return true;
 }
